# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/mochitest/mochitest.ini
# Commit: e9fc94b33da9
# Full Hash: e9fc94b33da9dc047b37e8bd2efe7f6b20fda079
# Author: Peter Van der Beken <peterv@propagandism.org>
# Date: 2021-12-14 21:34:09
# Regressor Bug: 1742865
# File Overlap Count: 1
# Description:
#   Bug 1742865 - Handle meta refresh correctly with session history in the parent. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132837
# ==============================================================================

diff -r f5bcc7acaf26 -r e9fc94b33da9 docshell/test/mochitest/mochitest.ini
--- a/docshell/test/mochitest/mochitest.ini	Tue Dec 14 10:10:32 2021 +0000
+++ b/docshell/test/mochitest/mochitest.ini	Tue Dec 14 10:13:58 2021 +0000
@@ -125,6 +125,10 @@
 support-files =
   file_bug1741132.html
 skip-if = toolkit == "android" && !sessionHistoryInParent
+[test_bug1742865.html]
+support-files =
+  file_bug1742865.sjs
+  file_bug1742865_outer.sjs
 [test_bug1743353.html]
 support-files =
   file_bug1743353.html